packages/react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -1328,6 +1329,11 @@ export class DetailsListBase extends React.Component<IDetailsListProps, IDetails
         ...this._columnOverrides[column.key],
       };
 
+      // eslint-disable-next-line deprecation/deprecation
+      if (!(baseColumn.isCollapsible || baseColumn.isCollapsable)) {
+        minimumWidth += getPaddedWidth(baseColumn, props);
```

Comment:
I'm not in love with keeping this but I think we need to keep track of whether the list as a whole is overflowing the available space so we can opt out of column resizing to replicate the previous behavior from the linked bug.